services/softlayer_dns_domain.go

```diff
@@ -0,0 +1,99 @@
+package services
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	datatypes "github.com/maximilien/softlayer-go/data_types"
+	"github.com/maximilien/softlayer-go/softlayer"
+)
+
+type softLayer_Dns_Domain_Service struct {
+	client softlayer.Client
+}
+
+func NewSoftLayer_Dns_Domain_Service(client softlayer.Client) *softLayer_Dns_Domain_Service {
+	return &softLayer_Dns_Domain_Service{
+		client: client,
+	}
+}
+
+func (sldds *softLayer_Dns_Domain_Service) GetName() string {
+	return "SoftLayer_Dns_Domain"
+}
+
+func (sldds *softLayer_Dns_Domain_Service) CreateObject(template datatypes.SoftLayer_Dns_Domain_Template) (datatypes.SoftLayer_Dns_Domain, error) {
+	// check required fields
```

Comment:
Is this a reminder for something not done. Please add code or remove comment since it's distracting.